### PR TITLE
select chosen media when opening media centre through renderer_media

### PR DESCRIPTION
**Pull request by vrcAlbert**

### CODE REVIEW

**Comment by Meroje** [comment]
:+1:

File: static/admin/novius-os/js/jquery.novius-os.js [hunk 1]
```diff
@@ -1003,15 +1003,16 @@ define('jquery-nos',
             nosMedia : function(data) {
 
                 data = data || {};
-                var contentUrls = {
-                        'all'   : 'admin/noviusos_media/appdesk?view=media_pick',
-                        'image' : 'admin/noviusos_media/appdesk?view=image_pick'
```
Comment (Foine) [inline static/admin/novius-os/js/jquery.novius-os.js:1008]: By set the url into only one variable, you loose the "view" information.

**Comment by smeeckaert** [comment]
Nice feature.

:+1:

File: static/admin/novius-os/js/jquery.novius-os.js (hunk 1 above)
Comment (vrcAlbert) [inline static/admin/novius-os/js/jquery.novius-os.js:1008]: I don't see why. It was not possible to change those config lines, so the only thing I did is to put this string into a single var (which still contains the "view" parameter, right before the "current_id").

Comment (vrcAlbert) [inline static/admin/novius-os/js/jquery.novius-os.js:1008]: Sorry, didn't read well the information :) I'll fix it!

**Comment by smeeckaert** [comment]
Nice feature :+1: since the config bug is fixed.

**Comment by jguyomard** [comment]
:+1:

**Comment by jguyomard** [comment]
:+1:

**Pull request merged**